parm/config/gfs/config.stage_ic

```diff
@@ -26,6 +26,12 @@ case "${CASE}" in
     export CPL_OCNIC=HR1_refactored
     export CPL_WAVIC=HR1_refactored
     ;;
+  "C1152")
+    export CPL_ATMIC=HR2_refactored
+    export CPL_ICEIC=HR1_refactored
+    export CPL_OCNIC=HR1_refactored
+    export CPL_WAVIC=HR1_refactored
+    ;;
```

Comment:
Since this is identical to C768, instead of repeating, please update line 23 as:
```
"C768" | "C1152")
```

Reply:
@aerorahul actually C1152 is aimed for HR3, so I am going to change them to HR3_refactored. Hos about that ?

Reply:
See below comment

Reply:
Ok.  Understood.  I was reviewing based on what is in the PR.  When you make the change, I will resolve this comment.